build/README.md

```diff
@@ -653,20 +638,22 @@ Since Minikube runs locally, there are some targets that need to be used instead
 
 #### `make minikube-test-cluster`
 Switches to an "agones" profile, and starts a kubernetes cluster
```

Comment:
I see that additional command which change profile was removed. Was it redundant?

Reply:
This is now powered by the `-p` argument in `minikube start` -  which also creates the profile. `minikube profile foo` will not create the profile if it doesn't exist.

https://minikube.sigs.k8s.io/docs/commands/start/